### PR TITLE
fix: Misaligned tooltip at the bottom of the privacy policy (Sanjeet4567)

### LINKED ISSUE #5443: Misaligned tooltip at the bottom of the privacy policy
### Did you clear cache before opening an issue?

- [X] I have cleared my cache

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Does the issue happen when logged in?

Yes

### Does the issue happen when logged out?

Yes

### Does the issue happen in incognito mode when logged in?

Yes

### Does the issue happen in incognito mode when logged out?

Yes

### Account name

unrealapex

### Account config

_No response_

### Current Behavior

When hovering over the tooltip, it is center aligned away from the the text being hovered over.

### Expected Behavior

The tooltip would appear right above Miodec's username.

### Steps To Reproduce

1. Go to the [contact info section of the privacy policy](https://monkeytype.com/privacy-policy#Contact_Info)
2. Hover over `@miodec`
3. See bug

### Environment

- OS:  `6.8.9-gentoo-dist`(Gentoo)
- Browser: Firefox
- Browser Version: Mozilla Firefox 125.0.3

### Anything else?

misaligned tooltip

### PR DESCRIPTION
**Pull request by Sanjeet4567**
### Description
Earlier :When hovering over the tooltip, it is center aligned away from the the text being hovered over.
Screenshot from 2024-05-26 11-07-43

After fix:The tooltip  appears right above Miodec's username.
Screenshot from 2024-05-26 11-06-38

### Checks

- [x] Check if any open issues are related to this PR; if so, be sure to tag them below.
Fixes #5443 
- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username inside parentheses at the end of the PR title

Closes #5443

### CODE REVIEW

**Comment by unrealapex** [comment]
I think having the text as `Discord: @miodec` would be more consistent with the formatting above.

File: frontend/src/privacy-policy.html [hunk 1]
```diff
@@ -435,7 +435,7 @@ <h1 id="Contact_Info">How to contact us</h1>
         Discord:
         <span
           aria-label="Click To Copy"
-          data-balloon-pos="up"
+          data-balloon-pos="up-left"
```
Comment (unrealapex) [inline frontend/src/privacy-policy.html:438]: extoplasm brought up that this change does not center the tooltip. See [their feedback](https://github.com/monkeytypegame/monkeytype/issues/5443#issuecomment-2134157665).

**Comment by Sanjeet4567** [comment]
I have added the asked changes ,kindly review.
After:
Screenshot from 2024-05-28 14-15-41

**Comment by Sanjeet4567** [comment]
I have made the asked changes .Kindly review.
After:
Screenshot from 2024-05-28 14-34-55

**Comment by Miodec** [comment]
Heres the actual fix: https://github.com/monkeytypegame/monkeytype/commit/898278b3eb8ef2137fb27656a5bbbdfcd1ea7bad

Thank you for your PR though.

**Comment by Miodec** [comment]
> I have made the asked changes .Kindly review. After: Screenshot from 2024-05-28 14-34-55

Haha, looks like we made the same change at the same time :D

**Comment by Sanjeet4567** [comment]
> Haha, looks like we made the same change at the same time :D

yeah ,No problem ,maybe next time I will be quick and better : ]

**Pull request closed without merging**